spec/runtime/require_spec.rb

```diff
@@ -98,6 +98,20 @@
     expect(err).to eq("ZOMG LOAD ERROR")
   end
 
+  it "displays a helpful message if the required gem throws an error" do
+    build_lib "faulty", "1.0.0" do |s|
+      s.write "lib/faulty.rb", "HI"
+    end
+
+    gemfile <<-G
+      path "#{lib_path}"
+      gem "faulty"
+    G
+
+    run "Bundler.require", expect_err: true
```

Comment:
you'll need to change this to `"expect_err" => true`, because we still have to support Ruby 1.8.7. :fearful: